extensions/INTEL/SPV_INTEL_fp_max_error.asciidoc

```diff
@@ -0,0 +1,135 @@
+SPV_INTEL_fp_max_error
+======================
+
+== Name Strings
+
+SPV_INTEL_fp_max_error
+
+== Contact
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Registry
+
+== Contributors
+
+* Shuo Niu, Intel
+* Daniel Zhang, Intel
+
+== Notice
+
+Copyright (c) 2023 Intel Corporation.  All rights reserved.
+
+== Status
+
+Final draft
```

Comment:
Minor, mentioning since it came up in Wednesday's teleconference: are we OK with the status being "Final draft" or would we rather this be something like "Shipping"?

```suggestion
* Shipping
```